shared/entity/type.go

```diff
@@ -388,7 +479,14 @@ func (identity) path() []string {
 	return []string{"auth", "identities", pathPlaceholder, pathPlaceholder}
 }
 
-type authGroup struct{}
+func (identity) apiMetricsURLPrefixes() []string {
+	// For /{version}/auth and /{version}/certificates endpoints.
+	return []string{"auth", "certificates", ".well-known"}
```

Comment:
The /1.0 prefix is trimmed in case it exists, if it doesn't the first path part is considered the prefix. So `/internal` and `/.well-known` are being counted as expected.

Reply:
Ah ok cool. Should we add `oidc` to this list for `auth` then as well?